common-pages/src/Setting.tsx

```diff
@@ -74,7 +74,7 @@ function Setting(props: IProps)  {
 		const e: string[] = [];
 		if (editnewSetting.Name == null || editnewSetting.Name.length === 0)
 			e.push('A Name is required')
-		if (editnewSetting.Name != null && editnewSetting.Name.length > 0 && allSettings.findIndex(s => s.Name == editnewSetting.Name && s.ID != editnewSetting.ID) > -1)
+		if (editnewSetting.Name != null && editnewSetting.Name.length > 0 && allSettings.findIndex(s => s.Name === editnewSetting.Name && s.ID !== editnewSetting.ID) > -1)
```

Comment:
should check if a setting by this Name exists even if different Capitalization. use .toLowercase()

Reply:
Should be resolved in most recent commit

Reply:
I think this will fail. try running it in the console.

Reply:
This should be fixed now